Dockerfile.assisted-service

```diff
@@ -5,6 +5,11 @@ COPY swagger.yaml .
 COPY tools/generate_python_client.sh .
 RUN chmod +x ./generate_python_client.sh && SWAGGER_FILE=swagger.yaml OUTPUT=/build ./generate_python_client.sh
 
+# TODO: Find a pure Python3 base image, rather than relying on the golang one
```

Comment:
Does `quay.io/ocpmetal/swagger-codegen-cli:2.4.15` have python? It sounds reasonable that both the generation and the packaging (build) of the Python client would happen in the same stage.

Reply:
Seems like it doesn't even have Bash, let alone Python3.